docs/conf.bak

```diff
@@ -0,0 +1,489 @@
+# -*- coding: utf-8 -*-
```

Comment:
Note that this is just the original file -- I kept it around for reference -- if we are good with the changes I'll remove the file from the PR before merging.